### DISCUSSION TITLE
Cypress v10 and Jenkins - how do I upgrade on a command line only remote server?

### DISCUSSION THREAD

**Discussion by edaldridge** [cypress-io/cypress#22120] [CI setup]
Hi

Following the recent update to v10 of Cypress, tests running on Jenkins are failing with the following error:
Your configFile is invalid: <path to config file>
It threw an error when required, check the stack trace below:
Error: Cannot find module 'cypress'

Locally Cypress is working fine and I was able to upgrade, but Jenkins is running remotely on a server with no GUI so I am not sure how to upgrade it.  Jenkins fetches the tests to run from a Bitbucket repository, I am assuming I need to remove/replace certain files there in order for the tests to run successfully but I am not sure where to start.  I'd appreciate some guidance if possible?

Thanks
Ed

**Comment by MariaDiamond** [comment]
It would be really nice to have a guidance on what to do now because our pipelines are failing.
If you are releasing new Cyrpess and it is so much changes which should be done on your machine, you should really release a step by step upgrade to 10.0 of Jenkins pipelines. Otherwise, it does not make sense. Our automation is failing because of it. I had to downgrade back to 9.7.latest.

**Comment by edaldridge** [comment]
Would be really good to get an answer to this question, 10 has been out for a while now and I can't seem to find any guides or documentation on how to do this.  When Cypress 10 was released it broke my CI and it took me a while to get it working again, I don't want to take the chance of damaging my CI again without some steps to follow
